source/extensions/filters/common/lua/protobuf_converter.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "source/common/protobuf/protobuf.h"
+
+#include "lua.hpp"
+
+namespace Envoy {
+namespace Extensions {
+namespace Filters {
+namespace Common {
+namespace Lua {
+
+/**
+ * Utility class for converting protobuf messages to Lua tables
+ */
+class ProtobufConverterUtils {
+public:
+  /**
+   * Convert a protobuf message to a Lua table
```

Comment:
Please document where the table is. Does this function allocate a table and put it on top of the lua stack? Or does it find an existing table on the lua stack?